core/src/commonMain/kotlin/com/saveourtool/processbuilder/utils/FileUtils.kt

```diff
@@ -0,0 +1,42 @@
+/**
+ * File containing utils for files
+ */
+
+@file:JvmName("FileUtilsJVM")
+
+package com.saveourtool.processbuilder.utils
+
+import io.github.oshai.KotlinLogging
+import okio.FileSystem
+import okio.Path
+import kotlin.jvm.JvmName
+
+expect val fs: FileSystem
+
+internal val fileUtilsLogger = KotlinLogging.logger { }
```

Comment:
why we need a logger here?

Reply:
It is used in `jvmMain` and `nativeMain` in order to log `myDeleteRecursively` function

Reply:
please add a comment about it that we create a logger which is used only in actual implementations (now it looks like a logger is initialized, but used outside this util method)

Reply:
Seems that `myDeleteRecursively` will be removed in next PR, so will not leave a comment for now